charts/minibroker/values.yaml

```diff
@@ -42,3 +42,13 @@ rbac:
     name: minibroker
     annotations: {}
     labels: {}
+
+# Optional override parameters for each of the supported charts.
+# If defined these params take precedence over user-supplied values.
+overrideChartParams:
+  mariadb: ~
+  mongodb: ~
+  mysql: ~
+  postgresql: ~
+  rabbitmq: ~
+  redis: ~
```

Comment:
Without reading the comments, it's hard to figure out what these are just by the name. Maybe we can break-down a little bit while reserving space for other configurations on each key. Something like:

```yaml
provisioning:
  mariadb:
    parameters:
      static: ~
  mongodb:
    parameters:
      static: ~
```

Reply:
I agree that it's not super obvious, but on the other hand the structure you suggest looks overly complex to allow for hypopthetical extensions in the future. Maybe a compromise could be what I put up in e831702, i.e.
```yaml
provisioning:
  mariadb:
    overrideParams: ~
  mongodb:
    overrideParams: ~
  mysql:
    overrideParams: ~
  postgresql:
    overrideParams: ~
  rabbitmq:
    overrideParams: ~
  redis:
    overrideParams: ~
```

Reply:
`overrideParams` sounds reasonable!